examples/src/fibonacci/fib2/air.rs

```diff
@@ -41,12 +44,12 @@ impl Air for FibAir {
 
     fn evaluate_transition<E: FieldElement + From<Self::BaseField>>(
         &self,
-        frame: &EvaluationFrame<E>,
+        frame: &Self::Frame<E>,
         _periodic_values: &[E],
         result: &mut [E],
     ) {
-        let current = frame.current();
-        let next = frame.next();
+        let current = frame.row(0);
```

Comment:
What about adding methods current and next for DefaultEvaluationFrame? I think in that way you can keep the examples and simple Air programs almost unchanged.

Reply:
Good point. I'll add these back in.